packages/expect/src/__tests__/asymmetricMatchers.test.ts

```diff
@@ -211,6 +211,14 @@ test('ObjectContaining throws for non-objects', () => {
   jestExpect(() => objectContaining(1337).asymmetricMatch()).toThrow();
 });
 
+test('ObjectContaining does not mutate the sample', () => {
```

Comment:
Maybe a comment with the issue link? I imagine otherwise it's pretty hard to understand in the future why this test was added ("of course it doesn't mutate, none of the matchers should do that")

Reply:
`git blame` covers that use case, no?

Reply:
I guess so, for some time, yeah :sweat_smile: